injectors/index.js

```diff
@@ -48,13 +48,38 @@ if (!VALID_PLATFORMS.includes(platform)) {
   }
 
   if (process.argv[2] === 'inject') {
-	try {
-		result = await main.inject(platformModule, platform);
-	} catch (e) {
-		// this runs if path generator crashes (app folder doesnt exist)
-		console.log(`${AnsiEscapes.RED}Platform you specified isn't installed on this device!${AnsiEscapes.RESET}\n\nList of valid platforms:\n${AnsiEscapes.GREEN}${VALID_PLATFORMS.map(x => `${x}`).join('\n')}${AnsiEscapes.RESET}`);
-		process.exit(process.argv.includes('--no-exit-codes') ? 0 : 1);
-	}
+    try {
+      if (!process.argv.find(x => VALID_PLATFORMS.includes(x.toLowerCase()))) {
+        for (const current of VALID_PLATFORMS) {
+          try
+          {
+            result = await main.inject(platformModule, current);
```

Comment:
Instead of just trying to inject, add a function to check if the platform exists without installing, then add a log saying installing to {platform}.